mmdet/datasets/transforms/loading.py

```diff
@@ -693,3 +693,73 @@ def __repr__(self):
         return self.__class__.__name__ + \
                f'(min_gt_bbox_wh={self.min_gt_bbox_wh}, ' \
                f'keep_empty={self.keep_empty})'
+
+
+@TRANSFORMS.register_module()
+class LoadPseudoAnnotations(BaseTransform):
```

Comment:
Should its name be LoadEmptyAnnotations? Pseudo annotation doesn't mean it's empty.